### PR TITLE
fix(download): Fix etag not md5 on all downloads

### PR DESCRIPTION
**Pull request by u9g**
md5 isnt always in the etag anymore so lets not use etag at all anymore

### CODE REVIEW

File: lib/launcher_download.js [hunk 1]
```diff
@@ -44,11 +43,12 @@ class LauncherDownload {
...
     if (this.versionsInfos[version]) { return Promise.resolve(this.versionsInfos[version]) }
     return this.getVersionsList()
-      .then(versionsList => {
+      .then(versionsList => new Promise((resolve, reject) => {
         const versionInfos = versionsList.versions.find(({ id }) => id === version)
         const versionUrl = versionInfos.url
-        return etagDownload(versionUrl, this.mcPath + '/versions/' + version + '/' + version + '.json')
-      })
+        const path = `${this.mcPath}/versions/${version}/${version}.json`
+        downloadFile(versionUrl, path, null, null).then(() => resolve(path)).catch(err => reject(err))
```
Comment (rom1504) [inline lib/launcher_download.js:50]: That's not right, you should just return
Why do you need to ignore the error ?

Comment (rom1504) [reply lib/launcher_download.js:50]: Ah no i see you're not ignoring it
Then why do you need to do new Promise ?

Comment (u9g) [reply lib/launcher_download.js:50]: We need to reaolve the path for the next .then, so I cant just return the download file.

Comment (rom1504) [reply lib/launcher_download.js:50]: So you can do something like return download(...).then (() => path)

Comment (u9g) [reply lib/launcher_download.js]: I understand now, done in the latest commit

**Comment by rom1504** [comment]
Can you do a release commit?

**Pull request merged**